index.html

```diff
@@ -1,57 +1,287 @@
+  </div>
+  <div class="date">
+  <ul>
+  <li>Canopy</li>
+  <li><img src="images/dot.png" alt=""></li>
+  <li>Back End Dev</li>
+
+  <li><img src="images/dot.png" alt=""></li>
+  <li>2015</li>
+
+  </ul>
+
+  </div>
+  <div class="summary">
+  <h2>A daily selection of privately personalized reads; no accounts or sign-ups required.</h2>
+  </div>
+
+  <div class="languages">
+
+
+  <ul>
+  <li>html</li>
+  <li>Css</li>
+  <li>javascript</li>
+
+  </ul>
+
+
+
+  </div>
+
+  <div class="see-project">
+  <button>See Project</button>
+  </div>
+
+  </div>
+
+
+  </section>
+  <br>
+  <br>
+  <br>
+  <section>
+
+  <div class="snapshot-container">
+
+  <div class="snapshot"><img src="images/Snapshoot Portfolio2.png" alt=""></div>
+  <div class="pheading">
+  <h1>Multi-Post Stories</h1>
+  </div>
+  <div class="date">
+  <ul>
+  <li>Canopy</li>
+  <li><img src="images/dot.png" alt=""></li>
+  <li>Back End Dev</li>
+
+  <li><img src="images/dot.png" alt=""></li>
+  <li>2015</li>
+
+  </ul>
+
+  </div>
+  <div class="summary">
+  <h2>A daily selection of privately personalized reads; no accounts or sign-ups required.</h2>
+  </div>
+
+  <div class="languages">
+
+
+  <ul>
+  <li>html</li>
+  <li>Css</li>
+  <li>javascript</li>
+
+  </ul>
+
+
+
+  </div>
+
+  <div class="see-project">
+  <button>See Project</button>
+  </div>
+
+  </div>
+
+
+  </section>
+
+  <br>
+  <br>
+  <br>
+  <section>
+
+  <div class="snapshot-container">
+
+  <div class="snapshot"><img src="images/Snapshoot Portfolio3.png" alt=""></div>
+  <div class="pheading">
+  <h1>Tonic</h1>
+  </div>
+  <div class="date">
+  <ul>
+  <li>Canopy</li>
+  <li><img src="images/dot.png" alt=""></li>
+  <li>Back End Dev</li>
+
+  <li><img src="images/dot.png" alt=""></li>
+  <li>2015</li>
+
+  </ul>
+
+  </div>
+  <div class="summary">
+  <h2>A daily selection of privately personalized reads; no accounts or sign-ups required.</h2>
+  </div>
+
+  <div class="languages">
+
+
+  <ul>
+  <li>html</li>
+  <li>Css</li>
+  <li>javascript</li>
+
+  </ul>
+
+
+
+  </div>
+
+  <div class="see-project">
+  <button>See Project</button>
+  </div>
+
+  </div>
+
+
+  </section>
+
+  <br>
+  <br>
+  <br>
+
+  <section>
+
+  <div class="snapshot-container">
+
+  <div class="snapshot"><img src="images/Snapshoot Portfolio4.png" alt=""></div>
+  <div class="pheading">
+  <h1>Multi-Post Stories</h1>
+  </div>
+  <div class="date">
+  <ul>
+  <li>Canopy</li>
+  <li><img src="images/dot.png" alt=""></li>
+  <li>Back End Dev</li>
+
+  <li><img src="images/dot.png" alt=""></li>
+  <li>2015</li>
+
+  </ul>
+
+  </div>
+  <div class="summary">
+  <h2>A daily selection of privately personalized reads; no accounts or sign-ups required.</h2>
+  </div>
+
+  <div class="languages">
+
+
+  <ul>
+  <li>html</li>
+  <li>Css</li>
+  <li>javascript</li>
+
+  </ul>
+
+
+
+  </div>
+
+  <div class="see-project">
+  <button>See Project</button>
+  </div>
+
+  </div>
+
+
+  </section>
+
+  <br>
+  <br>
+  <br>
+
+  <section>
```

Comment:
- [ ] kindly add the missing border-radius in the top right corner of your About me section 

![image](https://user-images.githubusercontent.com/89282221/192071115-4e4baeaf-917a-4dd5-801a-50ed20ba7a83.png)